e2e/test/helpers/RawQuerySerializationClass.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Azure.Devices.E2ETests.Helpers
     /// </summary>
     public class RawQuerySerializationClass
     {
-        [JsonProperty(PropertyName = "TotalNumberOfDevices")]
```

Comment:
One does not need to specify the property name explicitly. It is the default first parameter.